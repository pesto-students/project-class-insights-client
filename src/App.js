import React, { Component } from 'react';

class App extends Component {
  render() {
    return (<h1>hello My world</h1>);
  }
}

export default App;

