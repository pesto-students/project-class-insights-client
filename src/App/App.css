/* Basic styles */
@import url('https://fonts.googleapis.com/css?family=Open+Sans:300i,400,600,700,800');
* {
  outline: none;
}

body {
  background: #fff;
  font-family: 'Open Sans', sans-serif;
  margin: 0;
  overflow-x: hidden;
  color: #67757c;
}

html {
  position: relative;
  min-height: 100%;
  background: #ffffff;
}

a:focus,
a:hover {
  text-decoration: none;
}

a.link {
  color: #455a64;
}

a.link:focus,
a.link:hover {
  color: #1976d2;
}

h1,
h2,
h3,
h4,
h5,
h6 {
  color: #455a64;
  font-weight: 400;
}

h1 {
  line-height: 40px;
  font-size: 36px;
}

h2 {
  line-height: 36px;
  font-size: 24px;
}

h3 {
  line-height: 30px;
  font-size: 21px;
}

h4 {
  line-height: 22px;
  font-size: 18px;
}

h5 {
  line-height: 18px;
  font-size: 16px;
  font-weight: 400;
}

h6 {
  line-height: 16px;
  font-size: 14px;
  font-weight: 400;
}

.text-white {
  color: #ffffff;
}

.text-danger {
  color: #ef5350;
}

.text-muted {
  color: #99abb4;
}

.text-warning {
  color: #ffb22b;
}

.text-success {
  color: #26dad2;
}

.text-info {
  color: #1976d2;
}

.text-inverse {
  color: #2f3d4a;
}

.text-blue {
  color: #02bec9;
}

.text-purple {
  color: #7460ee;
}

.text-primary {
  color: #5c4ac7;
}

.text-dark {
  color: #67757c;
}

.text-themecolor {
  color: #1976d2;
}

.input-group-addon [type=checkbox]:checked,
.input-group-addon [type=checkbox]:not(:checked),
.input-group-addon [type=radio]:checked,
.input-group-addon [type=radio]:not(:checked) {
  position: initial;
  opacity: 1;
}

.invisible {
  visibility: hidden;
}

a,
button {
  outline: none;
  text-decoration: none;
  color: #99abb4;
  transition: all 0.2s ease 0s;
}

a.active,
button.active,
a:focus,
button:focus,
a:hover,
button:hover {
  color: #252525;
  outline: none;
  text-decoration: none;
}

button:hover,
button:active,
button:focus {
  color: #fff;
}

ul {
  padding: 0;
  margin: 0;
}

li {
  list-style: none;
}

p {
  font-family: 'Poppins', sans-serif;
  color: #99abb4;
}

.h2,
.h3,
.h4,
.h5,
.h6,
h1,
h2,
h3,
h4,
h5,
h6 {
  color: #455a64;
}

/*    Colors
-------------------*/

.color-primary,
.text-primary {
  color: #4680ff;
}

.color-success,
.text-success {
  color: #26dad2;
}

.color-info,
.text-info {
  color: #62d1f3;
}

.color-danger,
.text-danger {
  color: #fc6180;
}

.color-warning,
.text-warning {
  color: #ffb64d;
}

.color-pink,
.text-pink {
  color: #e6a1f2;
}

.color-dark,
.text-dark {
  color: #444c67;
}

.color-grey,
.text-grey {
  color: #ddd;
}

/*    Background
--------------------------*/

.bg-primary {
  background: #4680ff;
  color: #ffffff;
  fill: #4680ff;
}

.bg-success {
  background: #26dad2;
  color: #ffffff;
  fill: #26dad2;
}

.bg-info {
  background: #62d1f3;
  color: #ffffff;
  fill: #62d1f3;
}

.bg-danger {
  background: #fc6180;
  color: #ffffff;
  fill: #fc6180;
}

.bg-warning {
  background: #ffb64d;
  color: #ffffff;
  fill: #ffb64d;
}

.bg-pink {
  background: #e6a1f2;
  color: #ffffff;
  fill: #e6a1f2;
}

.bg-dark {
  color: #ffffff;
  fill: #444c67;
}

.bg-transparent {
  background: transparent;
  color: #252525;
}

button.dropdown-item {
  cursor: pointer;
}

a.button-anchor {
  -webkit-appearance: inherit;
  appearance: inherit;
}

/* Loader CSS */
.loader {
  display: block;
  height: 20%;
  width: 10%;
  top: 48%;
  position: fixed;
  left: 50%;
}

.dot {
  width: 24px;
  height: 24px;
  background: #3ac;
  border-radius: 100%;
  display: inline-block;
  animation: slide 1s infinite;
}
.dot:nth-child(1) {
  animation-delay: 0.1s;
  background: #32aacc;
}
.dot:nth-child(2) {
  animation-delay: 0.2s;
  background: #64aacc;
}
.dot:nth-child(3) {
  animation-delay: 0.3s;
  background: #96aacc;
}
.dot:nth-child(4) {
  animation-delay: 0.4s;
  background: #c8aacc;
}
.dot:nth-child(5) {
  animation-delay: 0.5s;
  background: #faaacc;
}
@-moz-keyframes slide {
  0% {
    transform: scale(1);
  }
  50% {
    opacity: 0.3;
    transform: scale(2);
  }
  100% {
    transform: scale(1);
  }
}
@-webkit-keyframes slide {
  0% {
    transform: scale(1);
  }
  50% {
    opacity: 0.3;
    transform: scale(2);
  }
  100% {
    transform: scale(1);
  }
}
@-o-keyframes slide {
  0% {
    transform: scale(1);
  }
  50% {
    opacity: 0.3;
    transform: scale(2);
  }
  100% {
    transform: scale(1);
  }
}
@keyframes slide {
  0% {
    transform: scale(1);
  }
  50% {
    opacity: 0.3;
    transform: scale(2);
  }
  100% {
    transform: scale(1);
  }
}

/* End Loader */