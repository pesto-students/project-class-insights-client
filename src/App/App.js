import React from 'react';

import appRoutes from '../Router/appRoutes';

class App extends React.Component {
  render() {
    return (
      <div className="jumbotron">
        {appRoutes}
      </div>
    );
  }
}

export default App;
