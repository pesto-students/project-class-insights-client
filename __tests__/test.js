import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from '../src/App';

describe('<App /> shallow rendering', () => {
  it('h1 contains correct text', () => {
    const wrapper = shallow(<App />);
    expect(wrapper.find('h1').text()).toBe('hello My world');
  });

  it('matches the snapshot', () => {
    const snaptree = shallow(<App />);
    expect(toJson(snaptree)).toMatchSnapshot();
  });
});
