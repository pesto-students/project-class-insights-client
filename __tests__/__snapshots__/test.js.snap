// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> shallow rendering matches the snapshot 1`] = `
<h1>
  hello My world
</h1>
`;
